pkg/state/state.go

```diff
@@ -434,15 +435,12 @@ var DefaultFetchOutputDirTemplate = filepath.Join(
 	"{{ or .Release.Version \"latest\" }}",
 )
 
-func (st *HelmState) ApplyOverrides(spec *ReleaseSpec) {
-	if st.OverrideKubeContext != "" {
-		spec.KubeContext = st.OverrideKubeContext
-	}
-	if st.OverrideNamespace != "" {
-		spec.Namespace = st.OverrideNamespace
-	}
-
+func (st *HelmState) ReFormatNeeds(spec *ReleaseSpec) []string {
 	var needs []string
+	releaseInstalledInfo := make(map[string]bool)
+	for _, r := range st.OrginReleases {
```

Comment:
What's the difference between `st.Releases` and `st.OriginReleases`?

I'd appreciate it if you could give me any pointer(file name + line number) that illustrates the biggest difference between the two.. 🙏